scripts/build.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+set -e
```

Comment:
another thing I have ended up having to add to some of our scripts is `set -o pipefail` because if a command fails but its pipeing to another command it wont be caught by set -e

Reply:
Good call - I added that as well